account_payment_sale/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2014-2016 Akretion (http://www.akretion.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+# @author Alexis de Lattre <alexis.delattre@akretion.com>
+
+{
+    'name': 'Account Payment Sale',
+    'version': '11.0.1.0.0',
+    'category': 'Banking addons',
+    'license': 'AGPL-3',
+    'summary': "Adds payment mode on sale orders",
+    'author': "Akretion, "
+              "Tecnativa, "
+              "Odoo Community Association (OCA)",
+    'website': 'https://github.com/OCA/bank-payment',
+    'depends': [
+        'sale',
+        'account_payment_partner',
+    ],
+    'conflicts': ['sale_payment'],
```

Comment:
@pedrobaeza is "conflicts" considered somewhere?

Reply:
No, it's only informative, and this doesn't make sense right now. It was for v7...